app/models/application_choice.rb

```diff
@@ -71,7 +71,7 @@ def submitted?
   end
 
   def decision_pending?
-    ApplicationStateChange::DECISION_PENDING_STATUSES.include? status.to_sym
+    (ApplicationStateChange::DECISION_PENDING_STATUSES + %i[inactive]).include? status.to_sym
```

Comment:
I wonder if we can't only have `DECISION_PENDING_STATUSES` here by adding `inactive` to the constant and replace `DECISION_PENDING_STATUSES` with `INTERVIEWABLE_STATES` in the codebase as the two constants are identical. And then we can use `decision_pending?` in `interview_bookings_component.html.erb`.

Reply:
I'm conscious about modifying the `ApplicationStateChange` constants as there's going to be a longer term piece of work to completely refactor this. (am happy to make the change though!) What're your thoughts on this @inulty-dfe ?

Reply:
Yeah go ahead, one more exception won't make a difference to the work that needs to be done.

Reply:
Adding a status to one of the constants is risky cause we don't know everywhere they're being used and it's not something I'd like to deal with in the middle of an unrelated PR. We need to sort out the status categories with full focus.

Reply:
That's why I said `replace DECISION_PENDING_STATUSES with INTERVIEWABLE_STATES in the codebase `. For whatever reason we have two identical constants.

Reply:
`INTERVIEWABLE_STATES` was added 3 years ago [here](https://github.com/DFE-Digital/apply-for-teacher-training/pull/4077/files). Maybe @duncanjbrown didn't realise an [identical constant](https://github.com/DFE-Digital/apply-for-teacher-training/blob/465de22e3c4793ca4b8e57c48665d12e1be04637/app/services/application_state_change.rb#L12C8-L12C8) was already present. Or maybe although the content was the same he wanted to emphasise some semantical differences. 🤔

Reply:
For the purpose of this PR I'm going to merge it with the current approach. The introduction of `inactive` – as a decision pending status, that also needs to be treated as a rejection – has introduced complexity into these state constants, and as Iain has mentioned this piece of work needs to be tackled with focus to understand the extent of their usage.

Reply:
When we do come to tackle it, we can audit all of them & remove any redundant constants 🧹

Reply:
Then add a 3rd constant that includes `inactive` 😄

Reply:
Ok – I don't mind doing that. Just need to think of a name now.... 🤔